### PR TITLE
[CrowdStrike] Add new filter: CROWDSTRIKE_REPORT_TARGET_INDUSTRIES

### PR DESCRIPTION
**Pull request by Lhorus6**
### Proposed changes

* Add a new filtering method : CROWDSTRIKE_REPORT_TARGET_INDUSTRIES

### Related issues

* https://github.com/OpenCTI-Platform/connectors/issues/2663

### Checklist

- [X] I consider the submitted work as finished
- [ ] I tested the code for its functionality using different use cases
- [X] I added/update the relevant documentation (either on github or on notion)
- [X] Where necessary I refactored code to improve the overall quality

### Further comments

To be review and tested. The filter is good, we just have to make sure that if we don't fill in this new variable, the connector is not broken.

### CODE REVIEW

File: external-import/crowdstrike/docker-compose.yml
Comment (yassine-ouaamou) [inline external-import/crowdstrike/docker-compose.yml:25]: Can you update the README file to add this parameter?

Comment (Lhorus6) [reply external-import/crowdstrike/docker-compose.yml:25]: I did it ✅

File: external-import/thehive/src/utils.py
Comment (Powlinett) [inline external-import/thehive/src/utils.py:1]: it should probably be removed from this PR as it's not in the scope of crowdstrike connector, right?

**Review by Powlinett — COMMENTED** [review]
✅ tested locally
✅ tested as Docker image
✅ in both cases, tested with and without any value for `CROWDSTRIKE_REPORT_TARGET_INDUSTRIES`env var

Code is clear, everything looks to work as intended, thanks @Lhorus6 🙂

Waiting for an answer to my comment and then it'll be good for me 😇

File: external-import/thehive/src/utils.py
Comment (Lhorus6) [reply external-import/thehive/src/utils.py:1]: I took the opportunity to correct a well-known deprecated function that has already been replaced in a number of places (i.e. in other connectors). 

If it's annoying, we can remove it. Otherwise, let's fix this.

TLDR: I would leave it, but if it's not a good practice and you don't want to, we can remove it.

**Comment by Lhorus6** [comment]
Thanks a lot, super fast @Powlinett 🫶 

I'll let you decide for your comment (see my reply)

File: external-import/thehive/src/utils.py
Comment (Powlinett) [reply external-import/thehive/src/utils.py:1]: It's not a good practice indeed, especially when it's not the same connector, because we need to maintain repo's history as clean/clear as possible in case of future investigations.
An issue has been created from this commit, it'll be merged soon so I'm deleting the file from this PR👍

Comment (Lhorus6) [reply external-import/thehive/src/utils.py:1]: Ok understood, so I won't do that in the future 👍

**Review by Powlinett — APPROVED** [review]

**Pull request merged**
